README.md

```diff
@@ -0,0 +1,8 @@
+# gluon-sargon
+Gluon packages from sargon
+
+## ddhcpd
+A distributed dhcp server. This package adds the ddhcpd from https://github.com/sargon/ddhcpd
```

Comment:
A distributed dhcp server. This package adds ddhcpd from https://github.com/sargon/ddhcpd